client/src/pages/Landing/Landing.tsx

```diff
@@ -0,0 +1,46 @@
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import { FormikHelpers } from 'formik';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+import FindSitter from './LandingForm/LandingForm';
+import { useAuth } from '../../context/useAuthContext';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import dogs from '../../Images/pets.jpg';
+
+export default function Landing(): JSX.Element {
+  const classes = useStyles();
+  const { updateLoginContext } = useAuth();
+  const { updateSnackBarMessage } = useSnackBar();
+
+  const handleSubmit = (
+    { address, startDate, endDate }: { address: string; startDate: Date; endDate: Date },
+    { setSubmitting }: FormikHelpers<{ address: string; startDate: Date; endDate: Date }>,
+  ) => {
+    //construct this function in the next ticket
+  };
+
+  return (
+    <Grid container component="main" className={classes.root}>
+      <CssBaseline />
+      <Grid className={classes.column}>
```

Comment:
The Grid component has two types - item/container. Please specify the appropriate type here and throughout :)

Reply:
I was told that Grid has item property by default last year. Today, I found that both item and container are false by default from MUI API docs. Actually, I use item here.